plugin/trino-faker/src/test/java/io/trino/plugin/faker/TestFakerQueries.java

```diff
@@ -192,8 +194,25 @@ void testSelectLimit()
         String testQuery = "SELECT count(rnd_bigint) FROM (SELECT rnd_bigint FROM single_column LIMIT 5) a";
         assertQuery(testQuery, "VALUES (5)");
 
+        testQuery = "SELECT count(rnd_bigint) FROM (SELECT rnd_bigint FROM single_column LIMIT %d) a".formatted(2*MAX_ROWS_PER_SPLIT);
+        assertQuery(testQuery, "VALUES (%d)".formatted(2*MAX_ROWS_PER_SPLIT));
```

Comment:
Add a space before & after `*`. Same for others.
```suggestion
        testQuery = "SELECT count(rnd_bigint) FROM (SELECT rnd_bigint FROM single_column LIMIT %d) a".formatted(2 * MAX_ROWS_PER_SPLIT);
        assertQuery(testQuery, "VALUES (%d)".formatted(2 * MAX_ROWS_PER_SPLIT));
```